app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="yuricfurusho.traderev">
 
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
```

Comment:
The addition of Glide library required this permissions, which we would eventually have added when accessing the Unsplash API.